scap/src/capturer/engine/win/mod.rs

```diff
@@ -66,29 +139,115 @@ impl WindowsCaptureHandler for Capturer {
 
 impl WinStream {
     pub fn start_capture(&mut self) {
-        // TODO: Prevent cloning the transmitter
+
         let capture_control = Capturer::start_free_threaded(self.settings.clone()).unwrap();
         self.capture_control = Some(capture_control);
     }
 
     pub fn stop_capture(&mut self) {
-        let mut capture_control = self.capture_control.take().unwrap();
-        capture_control.stop();
+        let capture_control = self.capture_control.take().unwrap();
+        let _ = capture_control.stop();
     }
 }
 
-pub fn create_capturer(options: &Options, tx: mpsc::Sender<Frame>) -> WinStream {
+#[derive(Clone, Debug)]
+struct FlagStruct {
+    pub tx: mpsc::Sender<Frame>,
+    pub crop: Option<CGRect>,
+}
+
+pub fn create_capturer(
+    options: &Options,
+    tx: mpsc::Sender<Frame>,
+) -> WinStream {
     let settings = Settings::new(
         Monitor::primary().unwrap(),
         Some(true),
         None,
-        ColorFormat::Rgba8,
-        tx,
-    )
-    .unwrap();
+        ColorFormat::Bgra8,
+        FlagStruct { tx, crop: Some(get_source_rect(options))},
+    
+    ).unwrap();
 
     return WinStream {
         settings,
         capture_control: None,
     };
 }
+
+pub fn get_output_frame_size(options: &Options) -> [u32; 2] {
```

Comment:
Do we not need the scale factor logic the way we have for mac? Is it possible to get the `scale` value in windows?